gpii/node_modules/couchConnector/src/couchConnector.js

```diff
@@ -0,0 +1,114 @@
+/*!
+
+    Convenience grade to provision and manage a CouchDB harness provided by the gpii-couchdb-test-harness package.
+    Intended as a replacement for the legacy "Pouch Manager".
```

Comment:
To future readers this reference to a thing which they have never heard of nor need to will seem baffling.

Reply:
Agreed.